docs/application/native/guides/multimedia/media-handle.md

```diff
@@ -7,11 +7,11 @@ The main features of the Media Tool API include:
 
 - Creating the media format handle
 
-  You can [set and get video or audio information](#format) using the `media_format_h` handle.
+  You can [set and get video or audio information](#format) using the `media_format_h` handle like average bps, bit, channel, channel mask, mime type, samplerate, width, height and frame rate for each type.(audio/video/text)
 
 - Creating the media packet handle
 
-  You can [set and get metadata](#packet) using the created `media_packet_h` handle. Some `media_packet_h` metadata are NOT filled after creating the `media_packet_h` handle: pts, dts, duration, extradata, codec data, codec data size, and buffer flags.
+  You can [set and get metadata](#packet) using the created `media_packet_h` handle. Some `media_packet_h` metadata are NOT filled after creating the `media_packet_h` handle: pts, dts, duration, extradata, codec data, codec data size, rotation method and buffer flags.
```

Comment:
Suggest rephrasing to:

You can [set and get metadata](#packet) using the created `media_packet_h` handle. However, some of the `media_packet_h` metadata such as pts, dts, duration, extra data, codec data, codec data size, rotation method, and buffer flags are not filled after creating the `media_packet_h` handle.

Please verify.
Also, is it extra data or extradata?

Reply:
Done.